test/software/amazon/ion/impl/bin/IonManagedBinaryWriterTest.java

```diff
@@ -104,6 +104,12 @@ else if (IonType.isContainer(type))
         }
     }
 
+    @Override
+    public int bvmLength() {
```

Comment:
The code revers to this as IVM in other parts, I'd prefer to keep this consistent.